daisy_workflows/image_import/enterprise_linux/translate.py

```diff
@@ -148,12 +115,21 @@ def DistroSpecific(g: guestfs.GuestFS):
   # Historically, translations have failed for corrupt dbcache and rpmdb.
   g.sh('yum clean -y all')
 
-  upgrade_ca_certificates(g)
-
   if install_gce == 'true':
     logging.info('Installing GCE packages.')
     g.write('/etc/yum.repos.d/google-cloud.repo', repo_compute % el_release)
     if el_release == '6':
+      # yum operations fail when the epel repo is used with stale
+      # ca-certificates, causing translation to fail. To avoid that,
+      # update ca-certificates when the epel repo is found.
+      #
+      # The `--disablerepo` flag does the following:
+      #  1. Skip the epel repo for *this* operation only.
```

Comment:
The old logic had a check (on error) that epel is used. The PR description mentions update step is done when epel is used. Where is this step performed in this PR?

Reply:
It's at line 129. The update will only occur if epel is installed; if it's not installed, the yum command will fail, and the update won't occur.